summit.html

```diff
@@ -306,6 +325,26 @@ <h2>Day 2 (Thursday, September 5)</h2>
               <td>9:30</td>
             </tr>
 
+            <tr>
+              <td>
+                <details>
+                  <summary>
+                    Eiffel Collector
+                  </summary>
+                  <div>
+                    <p>
+                      TBD
+                    </p>
+                    <p>
+                      <b>Speaker:</b> Tobias Åkervik (Ericsson)
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-error">Intermediate</span></td>
```

Comment:
I think the border will render red, right?
```suggestion
              <td><span class="tag bd-success" style="border: 1px solid blue !important;">Intermediate</span></td>
```

Reply:
Yes, doh. Done.